### ISSUE TITLE
Reintroduce noscript-wrapped redirects blocking

### ISSUE THREAD

**Issue by KOLANICH** [hackademix/noscript#226]
Some websites intentionally disrupt workflows of users without JS.
For example `vk.com` redirects any user without JS to [the page](https://vk.com/badbrowser.php) advertising installation of a "modern browser", promoting the own fork of Chromium first of all.

I remember that in XUL/XPCOM-based versions of NoScript there was a feature to disallow redirects wrapped into `noscript` tags.

**Comment by hackademix** [comment]
Yes, this feature existed in Classic.
At this moment you can partially make up for it either by disabling the noscript capability (for DEFAULT or CUSTOMizing the site) or by having Firefox to ask for confirmation for meta refresh (**accessibility.blockautorefresh** _about:config_ preference set to **false**).

**Comment by KOLANICH** [comment]
>At this moment you can partially make up for it either by disabling the noscript capability (for DEFAULT or CUSTOMizing the site)

Yeah, I currently have to resort on this. The sites doing such redirects usually don't use `<noscript>` tags for anything useful for such users. The problem with that is  that I cannot return to the page I had been redirected from by just using `back` feature, I return to the page before that one and it is very inconvenient.

**Comment by shadow7483147** [comment]
Very annoying for sites that do this. Is there a userscript I can use to prevent \<noscript\> from being used for specific websites in Chromium? Using ungoogled-chromium and Violentmonkey.
